oggm/core/climate.py

```diff
@@ -110,87 +110,39 @@ def process_custom_climate_data(gdir):
     gdir.write_pickle(out, 'climate_info')
 
 
-@entity_task(log, writes=['cesm_data'])
-def process_cesm_data(gdir, filesuffix='', fpath_temp=None, fpath_precc=None,
-                      fpath_precl=None):
-    """Processes and writes the climate data for this glacier.
-
-    This function is made for interpolating the Community
-    Earth System Model Last Millenial Ensemble (CESM-LME) climate simulations,
-    from Otto-Bliesner et al. (2016), to the high-resolution CL2 climatologies
-    (provided with OGGM) and writes everything to a NetCDF file.
+@entity_task(log, writes=['gcm_data'])
+def process_gcm_data(gdir, filesuffix='', precp=None, temp=None,
```

Comment:
For consistency with the rest of the code, can you call this `prcp` instead of  `precp`?